doc/introduction/new_opmath.rst

```diff
@@ -0,0 +1,161 @@
+.. _new_opmath:
+
+New opmath
+==========
+
+Version ``0.36`` of PennyLane updated the default ``Operator`` subclasses, changing the operator arithmetic.
+An end-user should not notice any (breaking) changes.
+This pages should help developers with troubleshooting and guide in the process of legacy support while both systems are supported.
+
+.. note::
+
+    If you are looking for a quick fix, jump to the :ref:`Troubleshooting` section!
+
+    After visiting the :ref:`Troubleshooting` section, if you are still stuck then you can:
+
+    - Post on the PennyLane `discussion forum <https://discuss.pennylane.ai>`_.
+
+    - If you suspect that your issue is due to a bug in PennyLane itself, please open a
+      `bug report <https://github.com/PennyLaneAI/pennylane/issues/new?labels=bug+%3Abug%3A&template=bug_report.yml&title=[BUG]>`_
+      on the PennyLane GitHub page.
+
+.. note::
+
+    The old return system has been completely removed from PennyLane as of version 0.33.0, along with the
+    ``enable_return()`` and ``disable_return()`` toggle functions.
+
+Summary of the update
+---------------------
+
+The opt-in feature ``qml.operation.enable_new_opmath()`` is now the default. You can still opt-out via
+``qml.operation.disable_new_opmath()``.
+
+The changes between the old and new system mainly concern Python operators ``+ - * / @``,
+that now create the following ``Operator`` subclass instances.
+
+
++----------------------------------+----------------------+---------------------------+

```

Comment:
any way to spice this up, looks a bit bland? 
![image](https://github.com/PennyLaneAI/pennylane/assets/43949391/2c040292-4a4c-4209-bf58-c6c6db416f2c)